crates/swc_plugin_runner/src/lib.rs

```diff
@@ -29,3 +33,14 @@ pub fn create_plugin_transform_executor(
 ) -> Result<TransformExecutor, Error> {
     TransformExecutor::new(path, cache, source_map, metadata_context, plugin_config)
 }
+
+#[cfg(not(feature = "__rkyv"))]
+pub fn create_plugin_transform_executor(
+    path: &Path,
+    cache: &Lazy<cache::PluginModuleCache>,
+    source_map: &Arc<SourceMap>,
+    metadata_context: &Arc<TransformPluginMetadataContext>,
+    plugin_config: Option<serde_json::Value>,
+) -> Result<TransformExecutor, Error> {
+    unimplemented!("Transform plugin cannot be used without serialization support")
```

Comment:
I was worried about this

Reply:
Is this really required?
I think it should not compile if there's no rkyv

Reply:
all of the imports are guarded by flags, this won't occur in normal condition. This was introduced to let plain `cargo check` does not fail due to missing exported signatures.

I'd like to remove it, but then all of CI / and `cargo check` will break right away.